KarhooUISDK.xcodeproj/project.pbxproj

```diff
@@ -3070,6 +3072,7 @@
 			children = (
 				099106FE21B83AF700BD7E6F /* KarhooSDKExtensions */,
 				0973872121DE77B3004BD5F9 /* NSAttributedString+Utils.swift */,
+				6F370A3A252C6E5100782411 /* TouchAreaEdgeInsets.swift */,
```

Comment:
ah, this was missing...

Reply:
yes I put it back and made sure it does build with it